dev-libs/libmcrypt/libmcrypt-2.5.8.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="Replacements for the old Unix crypt"
+DESCRIPTION="libmcrypt are intended to be replacements for the \
+old Unix crypt, except that they are under the GPL and \
+support an ever-wider range of algorithms and modes."
+HOMEPAGE="http://mcrypt.sourceforge.net/"
+COPYRIGHT="1998-2002 Nikos Mavroyanopoulos"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/mcrypt/libmcrypt-$portVersion.tar.gz"
+CHECKSUM_SHA256="e4eb6c074bbab168ac47b947c195ff8cef9d51a211cdd18ca9c9ef34d27a373e"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libmcrypt$secondaryArchSuffix = $portVersion
+	lib:libmcrypt$secondaryArchSuffix = $portVersion
```

Comment:
We use:
* `$portVersion` for `libmcrypt` and `libmcrypt_devel`
* `$libVersionCompat` for `lib:libmcrypt`
* `$libVersion` for `devel:libmcrypt`

And even if $libVersion is the same as $portVersion, we define it with `libVersion="$portVersion"`.

```diff
+libVersion="$portVersion"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
 PROVIDES="
 	libmcrypt$ secondaryArchSuffix= $portVersion
-	lib:libmcrypt$secondaryArchSuffix = $portVersion
+	lib:libmcrypt$secondaryArchSuffix = $libVersionCompat
 	"
```
We only know if libVersion is portVersion after a compilation (or if we read the Makefiles).

When you call haikuporter, you can use the `--strict-policy` option (same as `-S`).